plugins/search/providers/elastic_search/lib/api/enum/KalturaESearchGroupFieldName.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * @package plugins.elasticSearch
+ * @subpackage api.enum
+ */
+class KalturaESearchGroupFieldName extends KalturaStringEnum

```

Comment:
Add class KalturaESearchBaseUserFieldName that will include all the const below and will be used by user +TYPE , and by group